src/main/java/interfaces/Human.java

```diff
@@ -0,0 +1,24 @@
+package interfaces;
+
+public class Human implements Comparable<Human> {
+    protected String name;
```

Comment:
why protected? for test?

Reply:
This data must not be able for changing from outside. The method setName() exists for this action.

Reply:
Why not private ?

Reply:
Because i thout its private XD I will handle it soon.

Reply:
Oh, It protected because I doesen't want to rewrite methods in Man class.